lib/srv/desktop/rdp/rdpclient/src/util.rs

```diff
@@ -58,6 +58,20 @@ pub fn vec_u8_debug(v: &[u8]) -> String {
     format!("&[u8] of length {}", v.len())
 }
 
+/// Log a debug message that's useful for writing tests,
+/// but which we don't want in the debug logs otherwise.
+/// Turn this on and off by changing the if statement
+/// in the macro body.
+#[macro_export]
+macro_rules! test_debug {
+    // debug!("a {} event", "log")
+    ($($arg:tt)+) => {
+        if false { // Note to reviewers: this should be false in any PRs
```

Comment:
Have you looked at using conditional compilation for this? It seems like you could use `if cfg!(test) { ... }` to accomplish this without needing to remember to flip the bool back all the time.

You could also consider defining a feature flag to enable it explicitly when e.g. testing in situ:
```rust
if cfg!(any(test, feature = "foo")) { ... }
```

Then it'll work both in tests and whenever compiled with `--features=foo`. Admittedly this might be more difficult given our build situation.